app/design/frontend/base/default/template/oyst/oneclick/oneclick.phtml

```diff
@@ -16,9 +16,21 @@
 <?php
 if ($this->isSupportedProduct()) :
     ?>
+    <div id="oyst-1click-button-wrapper" style="
```

Comment:
@rus-daniel this should be in block like that we can test ```button_width``` and ```button_right_position```and if they are not setted, no need to display empty div ```oyst-1click-button-wrapper```